### ISSUE TITLE
Error on importing to BurpSuite

### ISSUE THREAD

**Issue by undefinedCody** [BitTheByte/BitMapper#2]
Hello.

I'm using BurpSuite professional 2023.2 and upon importing your script, I receive the following error:

```
SyntaxError: ('no viable alternative at input \'"Content-Length: {str(content_length)}"\'', ('C:\\Users\\Ace\\Downloads\\BitMapper.py', 32, 26, '            headers[i] = f"Content-Length: {str(content_length)}"\n'))

	at org.python.core.ParserFacade.fixParseError(ParserFacade.java:95)
	at org.python.core.ParserFacade.parse(ParserFacade.java:190)
	at org.python.core.Py.compile_flags(Py.java:2249)
	at org.python.core.__builtin__.execfile_flags(__builtin__.java:527)
	at org.python.util.PythonInterpreter.execfile(PythonInterpreter.java:287)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:577)
	at burp.iz9.h(Unknown Source)
	at burp.ip3.O(Unknown Source)
	at burp.k47.u(Unknown Source)
	at burp.e0u.lambda$panelLoaded$0(Unknown Source)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)

```

Could you please help me fix it?
Thanks

**Comment by CarlesLlobet** [comment]
Should be fixed with PR #4. Could you confirm @undefinedCody ?

**Comment by undefinedCody** [comment]
@CarlesLlobet , Hi. I'm using Burp Suite Professional v2023.10.3.7 now and the issue appears to be solved.
Thank you
